hpx/runtime/threads/policies/static_queue_scheduler.hpp

```diff
@@ -29,6 +29,8 @@
 
 #include <hpx/config/warnings_prefix.hpp>
 
+// JB set work stealing false in mode set
+
```

Comment:
Leftover comment?

Reply:
yes. Fixed in upcoming commit.
Existing schedulers override get_scheduler_mode to set the stealing flags to false if they do not support it. I have changed to to override the set_xxx instead and left that comment to remind me to check all schedulers are doing the right thing with flag they do not supoort

Reply:
Remove?